test/unit/annotation_spec.js

```diff
@@ -2340,6 +2340,49 @@ describe("annotation", function () {
         })
         .catch(done.fail);
     });
+
+    it("should handle push buttons", function (done) {
+      const buttonWidgetRef = Ref.get(124, 0);
+      buttonWidgetDict.set("Ff", 0x10000);
```

Comment:
(Here and below) Note that we have constant for this, namely `AnnotationFieldFlag.PUSHBUTTON`, that we can use here instead of the magic number.